src/ar_miner_log.erl

```diff
@@ -46,7 +46,8 @@ watchdog() ->
 	after ?FOREIGN_BLOCK_ALERT_TIME ->
 		log(
 			"WARNING: No foreign blocks received from network. "
-			"Please check your internet connection."
+			"Please check your internet connection and port forwarding settings."
+			"Consult your ISP or cloud provider to forward incoming HTTP requests with port 1984 to your machine."
```

Comment:
We also need to keep polling mode in mind.

My suggested message for polling mode:
```
WARNING: No foreign blocks received from network. Please check your internet connection and the logfile for errors.
```

My suggestion for non-polling mode:
```
WARNING: No foreign blocks received from network. Please check that your node is available on port 1234 on your Internet IP address. E.g. browse to http://[IP address]:1234/info. If so, please check the logfile for errors.
```